components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -362,10 +381,12 @@ end
 
 function Import._mergePlacements(lpdbEntries, placements)
 	for placementIndex, lpdbPlacement in ipairs(lpdbEntries) do
-		placements[placementIndex] = Import._mergePlacement(
-			lpdbPlacement,
-			placements[placementIndex] or Import._emptyPlacement(placements[placementIndex - 1], #lpdbPlacement)
-		)
+		if placements[placementIndex] then
```

Comment:
did you check if this breaks functionality of adding empty slots where they are needed?

Reply:
It will

Reply:
well if this pr breaks functionality it should never be merged